types/resource.go

```diff
@@ -200,6 +200,7 @@ const (
 
 const (
 	NodeConditionReasonManagerPodDown            = "ManagerPodDown"
+	NodeConditionReasonPhysicalNodeDown          = "PhysicalNodeDown"
```

Comment:
`KubernetesNode` instead of `PhysicalNode`.

Normally the `physical` in the cloud computing meaning the bare metal machine.